partiql-ast/src/main/kotlin/org/partiql/ast/Ast.kt

```diff
@@ -188,8 +194,15 @@ public object Ast {
         return ExprPosition(lhs, rhs)
     }
 
+    // query set variant with only required field; optional fields set to null
     @JvmStatic
-    public fun exprQuerySet(body: QueryBody, orderBy: OrderBy?, limit: Expr?, offset: Expr?): ExprQuerySet {
+    public fun exprQuerySet(body: QueryBody): ExprQuerySet {
+        return ExprQuerySet(body, null, null, null)
+    }
+
+    // query set variant with all fields and nullable defaults set to null
+    @JvmStatic
+    public fun exprQuerySet(body: QueryBody, orderBy: OrderBy? = null, limit: Expr? = null, offset: Expr? = null): ExprQuerySet {
```

Comment:
(self-review) similarly for query set, create the most common use case of required field only since there are a few nullable arguments. All of the other (besides SFW) will specify null defaults and will use `@JvmOverloads`